vars/shared-CKANTest.var.yml

```diff
@@ -170,7 +170,7 @@ extensions:
 
     CKANExtQGOV:
       <<: *CKANExtQGOV
-      version: "QOL-7905-ckan-2.9"
```

Comment:
Why is CKANTest on master branches instead of develop?

Reply:
CKANTest is a sandbox, and gets manually altered as needed, but I set this as the default so it could be a useful check that all the `master` branches integrate properly.